src/document.ts

```diff
@@ -8,14 +8,18 @@ import * as utils from './utils';
 const DEFAULT_RENDERER = 'njk';
 const DEFAULT_VIEW = '/views/base.njk';
 
+interface DocumentParts {
+  body?: string | null;
+  fields?: any | null; // TODO: See if we can limit this.
+}
+
 export class Document {
   path: string;
   pod: Pod;
   renderer: Renderer;
   locale: Locale;
   readonly ext: string;
-  private _fields: any; // TODO: See if we can limit this.
-  private _body: string | null;
+  private _parts: DocumentParts;
```

Comment:
I don't think Google's typescript style guide suggests using the `_` prefix anymore for private members (using the `private` keyword should be enough).

Reply:
Ok, then we need to figure out other names for these since there is a `get content` that would conflict with the private variable name. Any suggestions on different names that are private but have a public accessor?